src/runtime/hexagon_remote/known_symbols.cpp

```diff
@@ -20,61 +20,61 @@ void *lookup_symbol(const char *name, const known_symbol *map) {
 extern "C" {
 
 // More symbols we need to support.
-extern int qurt_hvx_lock;
```

Comment:
Why does this need to change? It's fine, just wondering why, and if it is unnecessary churn. The overload resolution casting below is also a result of this.

Reply:
The previous definitions caused mismatch errors with the headers in the newer SDKs (likely starting with 4.0.0)...they were updated to match.

Reply:
What is the error you are seeing?

Reply:
Any of the symbols that are actually functions report errors that look like:

```
known_symbols.cpp:35:12: error: redefinition of 'sqrtf' as different kind of symbol
extern int sqrtf;
           ^
/local/scratch/dpalermo/TSDK/Hexagon_SDK/4.3.0.0/tools/HEXAGON_Tools/8.4.11/Tools/bin/../target/hexagon/include/math.h:484:7: note: previous definition is here
float sqrtf(float) _NO_THROW;
      ^
```
The history on this files shows we had to make this change internally once we started using Hexagon tools 8.4.x.  Looks like improved/tighter error checking in clang++.

Reply:
This would have been an error at any time in clang I think. I think the problem is a newer SDK started including these symbols transitively from the headers we include here when they weren't included before. But that's reason enough to change it I guess.